sematic/api_client.py

```diff
@@ -754,29 +754,42 @@ def _raise_for_response(
     exception: Optional[Exception] = None
     url, method = response.url, response.request.method
 
+    error_message = None
+    could_load_json = False
+    try:
+        response_json = response.json()
+        could_load_json = True
+        error_message = response_json.get("error", None)
+        if error_message is not None:
+            error_message = f"The server provided the error message: {error_message}. "
```

Comment:
Please move the trailing space from this string after instances of `{error_message}` in the multi-line messages below.
```suggestion
            error_message = f"The server provided the error message: {error_message}."
```

Reply:
done